tests/unit-tests/course-theme/test-class-sensei-course-theme-option.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * This file contains the Sensei_Course_Theme_Option_Test class.
+ *
+ * @package sensei
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Tests for Sensei_Course_Theme_Option class.
+ *
+ * @group course-theme
+ */
+class Sensei_Course_Theme_Option_Test extends WP_UnitTestCase {
```

Comment:
Added tests for the new method I created. This is my first test class (basically copy-pasted from another), let me know if I missed anything important!

Reply:
Looks great to me!

I just noticed a PHPCS and test fail because of the same reason: The types on the class attributes.
Besides that, the only thing I see is that we could add PHPDoc in all the tests to keep consistency. ;)

Reply:
Good! Thanks I had 7.4 as target PHP version in PHPStorm. Changed to 7.3 :)